### ISSUE TITLE
Schematic Design 

### ISSUE THREAD

**Issue by mikeshams** [lokeshchinthakuntla/Project#2]
Kunjalben - What issues did you face when working with EasyEDA?

**Comment by kunjalpatel2894** [comment]
I faced many issue while searching for a particular model in Easy EDA. I did multiple research to find a component especially when I was looking for a particular model. For example, When I was searching for ESP 8266 Wi-Fi module, I got results of so many wi-fi modules but the one I was exactly looking for. I looked up for almost 10/12 models of ESP 8266. After that, I got to see the model I am using in my project. One more thing I would like to mention that It is hard to find the footprint of the components in Easy EDA.

**Comment by kunjalpatel2894** [comment]
One more issue I faced during using Easy EDA was connecting components. When I clicked on the wire tool, and as I I started connecting the components, wires were not place properly. I watched videos of that on the internet about "how to place wire properly in Easy EDA" and practiced a little about connecting components. I had to connect 5 components with each other so it was very crucial for me to place wires properly and connect them.
